spec/unit/typesafe/config/token_spec.rb

```diff
@@ -0,0 +1,186 @@
+require 'spec_helper'
+require 'hocon'
+require 'test_utils'
+require 'pp'
+
+
+describe Hocon::Impl::Token do
+  Tokens = Hocon::Impl::Tokens
+
+  ####################
+  # Equality
```

Comment:
I'm :+1: on this test re-organization. I think when we start getting into the more meaty and complicated test files, however, we should try to keep things as close to the ruby code as possible in terms of organization as per @cprice404's suggestion.